game/core/game_map.py

```diff
@@ -67,14 +67,28 @@ def load(self):  # type: ignore[no-untyped-def]
 
         self.properties = my_map.properties
 
-        # Any layer with '_blocking' in it, will be a wall
         self.scene.add_sprite_list("wall_list", use_spatial_hash=True)
+
+        if not reverse_movement:
```

Comment:
Maybe this could be named `inverese_movement`? Or `moving_on_water`?

Reply:
Good idea, I like `inverse_movement`!